src/Roslyn.Diagnostics.Analyzers/UnitTests/BlankLines/BlankLinesBetweenStatementsTests_CSharp.cs

```diff
@@ -0,0 +1,1006 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Roslyn.Diagnostics.CSharp.Analyzers.BlankLines;
+using Test.Utilities;
+using Xunit;
+
+namespace Roslyn.Diagnostics.Analyzers.UnitTests.BlankLines
+{
+    using Verify = CSharpCodeFixVerifier<
+        CSharpBlankLinesBetweenStatementsDiagnosticAnalyzer,
+        CSharpBlankLinesBetweenStatementsCodeFixProvider>;
+
+    public class BlankLinesBetweenStatementsTests_CSharp
+    {
+        [Fact]
+        public async Task TestNotAfterPropertyBlock()
+        {
+            var code =
+@"
+class C
+{
+    int X { get; }
+    int Y { get; }
+}";
+
+            await new Verify.Test()
```

Comment:
I would not test the `FixedCode` when we don't expect a diagnostic to be raised. If you go with this change, it applies to some other tests.

Reply:
I disagree... i'm trying to doc expected behavior here about what is/isn't controlled by this analyzer.  this is very normal IMO for full testing of these types.